src/Umbraco.Core/Services/ContentEditingServiceBase.cs

```diff
@@ -250,7 +286,7 @@ private async Task<Attempt<ContentValidationResult, ContentEditingOperationStatu
             : Attempt.FailWithStatus<TContent?, ContentEditingOperationStatus>(ContentEditingOperationStatus.CancelledByNotification, null);
     }
 
-    private Attempt<TContent?, ContentEditingOperationStatus> OperationResultToAttempt(TContent? content, OperationResult? operationResult)
+    protected internal Attempt<TContent?, ContentEditingOperationStatus> OperationResultToAttempt(TContent? content, OperationResult? operationResult)
```

Comment:
If I'm not mistaken, this can go back to being `private` now?

Reply:
done 💪